client/my-sites/scan/wpcom-scan-upsell.tsx

```diff
@@ -20,19 +21,120 @@ import PromoCard from 'components/promo-section/promo-card';
 import PromoCardCTA from 'components/promo-section/promo-card/cta';
 import SidebarNavigation from 'my-sites/sidebar-navigation';
 import useTrackCallback from 'lib/jetpack/use-track-callback';
+import { preventWidows } from 'lib/formatting';
+import SecurityIcon from 'components/jetpack/security-icon';
 
 /**
  * Asset dependencies
  */
 import JetpackScanSVG from 'assets/images/illustrations/jetpack-scan.svg';
 import './style.scss';
 
-export default function WPCOMScanUpsellPage(): ReactElement {
+const ScanMultisiteBody: FunctionComponent = () => (
+	<PromoCard
+		title={ preventWidows( translate( 'WordPress multi-sites are not supported' ) ) }
+		image={ <SecurityIcon icon="info" /> }
+		isPrimary
+	>
+		<p>
+			{ preventWidows(
+				translate(
+					"We're sorry, Jetpack Scan is not compatible with multisite WordPress installations at this time."
+				)
+			) }
+		</p>
+	</PromoCard>
+);
+
+const ScanVPActiveBody: FunctionComponent = () => {
+	const onUpgradeClick = useTrackCallback( undefined, 'calypso_jetpack_scan_vaultpress_click' );
+	return (
+		<PromoCard
+			title={ preventWidows( translate( 'Your site has VaultPress' ) ) }
+			image={ <SecurityIcon icon="info" /> }
+			isPrimary
+		>
+			<p>
+				{ preventWidows(
+					translate(
+						'Your site is already protected by VaultPress. You can find a link to your VaultPress dashboard below.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Your site already is protected by VaultPress. You can find a link to your VaultPress dashboard below.' )` *ES Score: 10*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/wpcom-upsell-multisites)

Reply:
Resolved in 836fb82.